docs/data/stashcache/install-origin.md

```diff
@@ -241,7 +242,7 @@ For example:
 You can use the special value `ANY` to indicate that the origin will serve data from any VO that puts data on it.
 
 In addition to the origin allowing a VOs via the `AllowedVOs` list,
-that VO must also allow the origin in its `DataFederations/StashCache/AllowedOrigins` list.
+that VO must also allow the origin in one of its `AllowedOrigins` lists in `DataFederation/StashCache/Namespaces`.
```

Comment:
I think this is getting to the point of requiring an example, especially since `DataFederation/StashCache/Namespaces` looks like a path and it may not be clear that it's in reference to:

```
DataFederation:
  StashCache:
    Namespaces:
...
```

That or a link to the vo-data page

Reply:
I have a link to the vo-data page in the next sentence

Reply:
🤦🏽